lib/utilities/run-command.js

```diff
@@ -18,6 +23,11 @@ module.exports = function run(/* command, args, options */) {
   }
 
   debug("running command=" + command + "; args=" + args + "; cwd=" + process.cwd());
+
+  if (isWindows && command === 'npm') {
+    return exec(command + ' ' + args.join(' '));
```

Comment:
Cannot say much about the Windows specifics here, as I am not aware of them. I guess we have to trust you on this!? 😉